### ISSUE TITLE
Some secrets being erased when they are set since last update. 

### ISSUE THREAD

**Issue by happyjamstack** [dotenvx/dotenvx#228]
When you _change_ an existing variable defined in the `.env` file with:
```bash
dotenvx set VAR_2 'val_2b' -f '.env'
```
then variables occurring after the variable set in the `.env` file will be removed.

`.env` file before:
```env
VAR_1=val_1
VAR_2=val_2
VAR_3=val_3
VAR_4=val_4
```
`.env` file after:
```env
VAR_1=val_1
VAR_2=val_2b
```

This bug appears in version `0.41.0` but not `0.40.0`. 

For now, you can get around the bug by having a seperate copy of the file and copy pasting it into your file whenever you want to update an existing variable(or copy the encrypted `.env`  from github).

**Comment by motdotla** [comment]
test here: https://github.com/dotenvx/dotenvx/pull/227/files#diff-c1b9252b42f23bff38ea8844a9bd571245c4e95864e7ecf6a4aaa9b31a14ad0cR252

will go out shortly

**Comment by motdotla** [comment]
released as `v0.43.1`. upgrade to that and you should be good to go.
